uniffi_bindgen/src/bindings/kotlin/templates/wrapper.kt

```diff
@@ -23,47 +23,23 @@ import com.sun.jna.Pointer
 import com.sun.jna.Structure
 import java.nio.ByteBuffer
 import java.nio.ByteOrder
-import java.util.concurrent.atomic.AtomicLong
-import java.util.concurrent.atomic.AtomicBoolean
-import java.util.concurrent.atomic.AtomicReference
-import java.util.concurrent.locks.ReentrantLock
-import kotlin.concurrent.withLock
 
+{%- for imported_class in self.imports() %}
+import {{ imported_class }}
+{%- endfor %}
+
+// The Rust Buffer and 3 templated methods (alloc, free, reserve).
 {% include "RustBufferTemplate.kt" %}
 
-{% include "RustBufferHelpers.kt" %}
+{% include "Helpers.kt" %}
```

Comment:
I'd like to get this to a point where `Helpers.kt` and `RustBufferTemplate.kt` had no additional templated code left (it's very close).